uniffi_bindgen/src/bindings/swift/templates/Template-Bridging-Header.h

```diff
@@ -40,11 +40,10 @@ typedef struct ForeignBytes
 } ForeignBytes;
 
 // Error definitions
-// Each error has an error code enum, and a struct
-typedef struct NativeRustError {
-    int32_t code;
-    char *_Nullable message;
-} NativeRustError;
+typedef struct RustCallStatus {
+    int8_t code;
+    RustBuffer errorBuf;
+} RustCallStatus;
 
 // ⚠️ Attention: If you change this #else block (ending in `#endif // def UNIFFI_SHARED_H`) you *must* ⚠️
 // ⚠️ increment the version in all instance of UNIFFI_SHARED_HEADER_V1 in this file.                   ⚠️
```

Comment:
I wasn't sure what this meant.  Should I change the name to `UNIFFI_SHARED_HEADER_V2`?

Reply:
Yes, I think we should - otherwise we might end up using the declarations from some other component where `NativeRustError` is defined an `RustCallStatus` is not.

Reply:
(The failure behaviour if we *don't* do this will be safe, e.g. the user will get an error like "`RustCallStatus` not defined" or "`NativeRustError` not defined". But still, we should provide a better error in this case, since they may not realize it's due to mixing versions of UniFFI).